lib/urlwatch/storage.py

```diff
@@ -141,6 +141,12 @@
             'to': '',
             'subject': '{count} changes: {jobs}'
         },
+        'xmpp': {
+            'enabled': False,
+            'sender': '',
+            'password': '',
```

Comment:
Should we support authentication via keyring like with SMTP? And an optional in-config provided `insecure_password` (to make it more obvious that storing the password of a primary account isn't a good idea)